data/avro-schema/src/main/resources/avro/net/corda/data/membership/db/response/query/PersistGroupPolicyResponse.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "record",
+  "name": "PersistGroupPolicyResponse",
+  "namespace": "net.corda.data.membership.db.response.query",
```

Comment:
Should this be in `net.corda.data.membership.db.response.query` instead to mirror the request package?